modules/tinymce/src/core/main/ts/delete/TableDelete.ts

```diff
@@ -66,27 +81,39 @@ const setCursorInCell = (editor: Editor, cell: SugarElement<HTMLTableCellElement
  * - part of a table and content outside is selected
  */
 const emptySingleTableCells = (editor: Editor, cells: SugarElement<HTMLTableCellElement>[], outsideDetails: Optional<OutsideTableDetails>): boolean => {
-  // Remove content from selected cells
-  cleanCells(cells);
+  const editorRng = editor.selection.getRng();
+  const cellsToClean = outsideDetails.bind(({ rng, isStartInTable }) => {
+    /**
+     * Delete all content outside of the table that is in the selection
+     */
```

Comment:
```suggestion
    // Delete all content outside of the table that is in the selection
```
This doesn't need to be a multiline comment. If it was meant to be one for visibility, then it shouldn't be an API docs comment (i.e shouldn't have `/**`)

Reply:
Actually, maybe this should be outside this/above the `const cellsToClean ...` part?

Reply:
I did this because there are two separate chunks of logic in here, happy to change if anyone objects

Reply:
I don't like that this seems to conflict with the comments below then? Can we merge them?